src/transformers/modeling_albert.py

```diff
@@ -533,6 +532,8 @@ def forward(
 
         """
 
+        output_attentions = output_attentions if output_attentions is not None else self.config.output_attentions
```

Comment:
@thomwolf  @LysandreJik @sshleifer - This is written in every forward function so that we have the logic as explained in the description